### ISSUE TITLE
When will v2.0 be released

### ISSUE THREAD

**Issue by superSun88** [crabbly/Print.js#598]
When will v2.0 be released

**Comment by erqiu-sj** [comment]
Haha, I'm also looking for the answer to this question

**Comment by erqiu-sj** [comment]
> 

Haha, I'm also looking for the answer to this question

**Comment by daphnesmit** [comment]
I dont think this i maintained. I want to start a maintained version of this in typescript with modern code. Pls help me maintain with prs :D and lets make this library great again! https://github.com/daphnesmit/prntr
